ru.py

```diff
@@ -131,6 +134,13 @@ def stem(word, inflections = []):
 				return word[:-len(inflection)]
 		return word
 
+def replace_superscripts(text):
+	replaces = dict(zip('¹²³⁴⁵⁶⁷⁸⁹⁰', '          '))
```

Comment:
Please simplify this code by using sth like `text = re.sub(r'[⁰¹²³⁴⁵⁶⁷⁸⁹]', '', text)`?

The '²' case looks like overfitting the replace rules for a particular transcript. Unless there's a huge number of audio transcript with '²' that must be replaced in your proposed way, I strongly prefer we remove this hack. Then you can inline
```python
superscripts = '⁰¹²³⁴⁵⁶⁷⁸⁹'
text = re.sub(f'[{superscripts}]', ' ', text)
```
into `normalize_text`